lib/Stickersheet.js

```diff
@@ -0,0 +1,116 @@
+var Document = require('html-document/lib/Document').Document;
+var KeplerHeader = require('./KeplerHTTPHeader');
+var _ = require('lodash');
+
+var Stickersheet = module.exports = function(trixel) {
+    
+    // This stickersheet scales based on dpi. Normally the dpi would be 600
+    // Though if the page isn't 8.5 / 11 inches, things need to be programmatically repositioned.
+
+    // Our fake HTML document
+    var document = new Document();
+
+    var pageWidthInches = 8.5; // Assumed to be inches
+    var pageHeightInches = 11; // Assumed to be inches
+    var dpi = 100; // Change to 600 for production or whatever the passed in dpi is (Passing in dpi not set up yet)
```

Comment:
Even better if we can configure this!  `stickerSheet` option could be an object with settings rather than a simple true/false.